ecosystem/sep-0024.md

```diff
@@ -1276,6 +1285,7 @@ There is a small set of changes when upgrading from SEP-6 to SEP-24.
 
 ## Changelog
 
+- `v3.1.0`: Add XLM support to SEP-24 ([#1363](https://github.com/stellar/stellar-protocol/pull/1363))
```

Comment:
I suggest removing the term XLM here and replacing it with native. We typically refer to the native asset simple as so, and not the specific pubnet network's name for its native asset.

```suggestion
- `v3.1.0`: Add native asset support to SEP-24 ([#1363](https://github.com/stellar/stellar-protocol/pull/1363))
```

Reply:
Fixed!